core/template.php

```diff
@@ -235,6 +235,11 @@ function tpl_filters($filter, $showtv)
     $smarty->assign('listcolumns', session_get('listcolumns'));
 }
 
+function adultcheck_for_video($video)
```

Comment:
unrelated?

Reply:
see line 306 needed for replace of create_function